front/src/applications/stdcm/components/SimulationReportStyleSheet.ts

```diff
@@ -0,0 +1,553 @@
+import { StyleSheet, Font } from '@react-pdf/renderer';
+
+import IBMPlexMonoBold from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Bold.ttf';
+import IBMPlexMonoMedium from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Medium.ttf';
+import IBMPlexMonoRegular from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-Regular.ttf';
+import IBMPlexMonoSemiBold from 'assets/simulationSheet/fonts/IBM-Plex-Mono/IBMPlexMono-SemiBold.ttf';
+import IBMPlexSansBold from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Bold.ttf';
+import IBMPlexSansItalic from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Italic.ttf';
+import IBMPlexSansMedium from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Medium.ttf';
+import IBMPlexSansRegular from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Regular.ttf';
+import IBMPlexSansSemiBold from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-SemiBold.ttf';
+
+Font.register({
+  family: 'IBM Plex Sans',
+  fonts: [
+    { src: IBMPlexSansRegular, fontWeight: 'normal' },
+    { src: IBMPlexSansMedium, fontWeight: 'medium' },
+    { src: IBMPlexSansSemiBold, fontWeight: 'semibold' },
+    { src: IBMPlexSansBold, fontWeight: 'bold' },
+    { src: IBMPlexSansItalic, fontStyle: 'italic' },
+  ],
+});
+
+Font.register({
+  family: 'IBM Plex Mono',
+  fonts: [
+    { src: IBMPlexMonoRegular, fontWeight: 'normal' },
+    { src: IBMPlexMonoMedium, fontWeight: 'medium' },
+    { src: IBMPlexMonoSemiBold, fontWeight: 'semibold' },
+    { src: IBMPlexMonoBold, fontWeight: 'bold' },
+  ],
+});
+
+export const fakeInformation = {
+  rcName: 'Super Fret',
+  rcPersonName: 'Jane Smith',
+  rcPhoneNumber: '01 23 45 67 89',
+  rcMail: 'john.doe@example.com',
+  path_number1: 'n°123456',
+  path_number2: 'n°987654',
+};
```

Comment:
Can you move this to SimulationReportSheet ? It's not related to style :)

Reply:
done